api/jobset/v1alpha2/jobset_webhook.go

```diff
@@ -29,7 +29,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
 	"sigs.k8s.io/jobset/pkg/util/collections"
-	shared "sigs.k8s.io/jobset/pkg/util/shared"
+	shared "sigs.k8s.io/jobset/pkg/util/placement"
```

Comment:
```suggestion
	placement "sigs.k8s.io/jobset/pkg/util/placement"
```

and maybe update ref just so it’s not confusing

Reply:
Woops, thanks for catching that, looks like I missed a few references to `shared` - not sure how that happened, I guess the "rename symbol" didn't work as I expected in this case